src/XRP/issued-currency-client.ts

```diff
@@ -783,10 +853,25 @@ export default class IssuedCurrencyClient {
       const rawSendMaxValue = (1 + transferFee / 100) * numericValue
       const numDigitsLeftOfDecimal = String(Math.ceil(rawSendMaxValue)).length
       const allowedDigitsRightOfDecimal = 15 - numDigitsLeftOfDecimal
-      const sendMaxValue = String(
+      const calculatedSendMaxValue = String(
```

Comment:
i know this is from before but it's more idiomatic to use `Number.toString` instead of the `String` constructor

also - `rawSendMaxValue.toFixed(allowedDigitsRightOfDecimal)` is already a string so i'm not sure you need to do this?

Reply:
ohh I think you're right -- thought it left it as a number but nah